lib/automaticupgrades/config.go

```diff
@@ -58,3 +64,18 @@ func IsEnabled() bool {
 func GetChannel() string {
 	return os.Getenv(automaticUpgradesChannelEnvar)
 }
+
+// GetUpgraderVersion returns the teleport upgrader version
+func GetUpgraderVersion(ctx context.Context) string {
+	if os.Getenv("TELEPORT_EXT_UPGRADER") == "unit" {
+		out, err := exec.CommandContext(ctx, teleportUpgradeScript, "version").Output()
+		if err != nil {
+			log.WithError(err).Debug("Failed to exec teleport-upgrade version command.")
```

Comment:
I would log the exact binary's path here
```suggestion
			log.WithError(err).Debug("Failed to exec /usr/sbin/teleport-upgrade version command.")
```

It could help with the expected location of the binary, in case it ever changes